hack/update-openapi-spec.sh

```diff
@@ -72,7 +77,7 @@ fi
 # Start kube-apiserver
 # omit enums from static openapi snapshots used to generate clients until #109177 is resolved
 kube::log::status "Starting kube-apiserver"
-kube-apiserver \
+KUBE_APISERVER_STRICT_REMOVED_API_HANDLING_IN_ALPHA=${STRICT_ALPHA_HANDLING} kube-apiserver \
```

Comment:
won't being strict here drop all the types from the next minor once master goes to alpha.0, which happens at release branch cut time *before* .0 release of the previous minor?

Reply:
Resolved in https://github.com/kubernetes/kubernetes/pull/130320#discussion_r1980305938. Strict will only happen in alpha but not alpha.0